ui/src/app/store/vlan/utils.ts

```diff
@@ -18,3 +22,51 @@ export const getVLANDisplay = (vlan: VLAN | null): string | null => {
     return vlan.vid.toString();
   }
 };
+
+/**
+ * Get the text full name for a VLAN.
+ * @param vlanId - A VLAN's id.
+ * @return A VLAN's full name.
+ */
+export const getFullVLANName = (
+  vlanId: VLAN["id"],
+  vlans: VLAN[],
+  fabrics: Fabric[]
+): string | null => {
+  const vlan = vlans.find(({ id }) => id === vlanId);
+  if (!vlan) {
+    return null;
+  }
+  const fabric = fabrics.find(({ id }) => id === vlan.fabric);
+  if (!fabric) {
+    return null;
+  }
+  return `${getFabricDisplay(fabric)}.${getVLANDisplay(vlan)}`;
+};
+
+/**
+ * Get the text for the link mode of the interface.
+ * @param nic - A network interface.
+ * @return The display text for a link mode.
+ */
+export const getDHCPStatus = (
```

Comment:
I think I wrote pretty much exactly this in `NetworkCardTable.tsx`. Would you mind replacing the functions there with these utils?

Reply:
Oh sure, there are a few minor differences, but easily changed, I'll update now.

Reply:
Updates pushed.